ReservacionVuelos/Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+# You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
+ruby '2.7.4'
+
+gem 'cocoapods', '~> 1.11', '>= 1.11.2'
```

Comment:
Why is Ruby's Gemfile in your project?

Reply:
React -native generated the document when we init the project from console